components/brave_wallet_ui/assets/svg-icons/ledger-logo.svg

```diff
@@ -0,0 +1,11 @@
+<svg width="93" height="25" viewBox="0 0 93 25" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This is an old logo of Ledger. I'm uploading the new one here.

[Ledger_logo_blue_horizontal.svg.zip](https://github.com/brave/brave-core/files/6741678/Ledger_logo_blue_horizontal.svg.zip)

Reply:
Gotcha! I'll let James on the design team know and see if he want to replace it 😊